### DISCUSSION TITLE
Test Article [test-article]

### DISCUSSION THREAD

**Discussion by rootEnginear** [rootEnginear/gh-discussion-cms#1] [General]
Basically is the same as my <https://github.com/rootEnginear/gh-issues-cms> but using discussion which seems more appropriate and non-intrusive way to publish an article.

Issue's issue is that it will plague <https://github.com/issues> page, which is an issue for me (pun intended), so I would like to see if discussion is more doable for this task.

Constraints approaching this approach:
- Slug: I choose to handle this with slug at the end of title.
- Post state: I use label for this.

Todo?
- [ ] Make this work
- [ ] OG Image Generation

Below is just random text for testing markdown engine.

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed vestibulum metus quis ipsum rhoncus iaculis sed ac magna. Nam imperdiet pulvinar vulputate. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Ut in velit ultrices diam porta placerat. Aliquam vitae eros fermentum, fringilla libero condimentum, sollicitudin orci.

## Doliturus omnia foret fassoque veniam

Lorem **markdownum**, num pressant grates ut pro in fatis montanum latius fontes. Et solutus sanguine remoliri titulum agmen flos ensem purpureusque cognoscere. Corpora corpore incerta, igne proelia poma, sic varios Lycaei. Circaea enim.

```
solid *= laptopFormula(1);
rpmCopyModem.smart = 5;
var ict_bar_encoding = operation;
eccDataSwappable += systemVisual.program(cell_namespace, page -
        directoryFlashLink - bootHertz);
```

Nec insula, adit, pugnam. Fastigia enim. Romuleos *mare dea dicta* membra patriaque **fortis**, pectusque quippe meo. Aurum nemo messoris cum surgit nostros peream metit magni quem pudore summum libera concidit. Cupies perdes ulla prodere posito, ab quas quem pars si.

1. Quis confugit
2. Illius molimine iungitur iterum
3. Videri quoque
4. Via ac trahi et tenere te ante
5. Puero orbi huic retia iunctior Iovem
6. Venit genetrix invida

## Mariti excussit neque

Arte nata mittere! At resilire **curae aliamve ferarum** dimovit certos sereno Averna.

> Est Arcas omnesque praescius victus sumptis nascentia dicentem, in incipiat
> patulos: sensus lutulenta inpune. Est auras socerum pendebat digna, intus
> colla [rursus freta cum](http://www.manibus.net/vellere), partem, solum. Sibi
> tetigit hi partim, elige adit Titania cecidere coniugis referam bracchia,
> nullasque, [minister et verba](http://nec-incepta.io/tetendi-tangendo),
> stetit. Cum intrare atros ratibus agrestes thyrsos fecit missos cum ambarum,
> faciat hic amissos nervis praesagia occidit domumque. Et quamvis prius
> bracchia, cape a Typhoea!

Diu tuos, si hoc nova manu dixit: solvo scelerata [referebam toto](http://pandione.net/). Sed ille omnes fuerat volucrisque frena stirpis dum [fuit](http://tot.io/diemque-quis.html), sed Pallas, [attollens](http://petebamus.com/), precor. [Suam aut nec](http://www.delapsam-iam.io/) arte dato mundum, [dabitur](http://dea-at.net/comitata.html)? Vix quod: morari nondum insanis est, pennis iuris aderisque ullum, est!

```
var loadAjax = -3;
if (-3) {
    bus_compile_desktop -= mbpsMonitorSwitch;
} else {
    irq = banner;
}
word.waveformPpi(secondary_antivirus_hover + 5 * directVoipRecycle,
        jfs_virtualization_d.rawStationCore(server_petaflops_rom,
        binaryOutput));
if (optic) {
    busProcessPersonal -= daemon_simm(ad, raid_document, firmwareGraphicMail);
    lanAluWin += dataWrapPublishing;
} else {
    spoofing_phreaking *= -5;
    bps_jsp_moodle = ajax.clob(5, dslWordart) + array;
    extension.titleKeyErgonomics = function_third(abend, mebibyte,
            bankruptcyDrive(design_mamp, cpl, memoryDomain));
}
```

Ponti fulmina osculaque corpore [oravique](http://www.ulixem.com/summaquelacrimis), crinita callidus repulsae mihi [ovaque](http://id-illud.net/). Iteratque condita nullo **sed carmen**, me sensisse *iras hospitis* dei veteres praecluditur quorum.
